### PR TITLE
Convert cython to vanilla python + numpy

### PR DESCRIPTION
**Pull request by mjs**
This completes #34, fixing up tests and deployment.

Using numpy for compression and decompression is faster than the previous numpy implementation and allows the package to be pure Python which simplifies deployment.

### CODE REVIEW

File: cptv/reader.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+            val = self._read_float32(s)
+        elif ftype == Field.TIMESTAMP:
+            micros = self._read_uint64(s)
+            try:
+                val = epoch + timedelta(microseconds=micros)
+            except OverflowError:
+                print("timestamp is broken - using default")
+                val = epoch
+        else:
+            # Unknown field, just slurp up the bytes
```
Comment (CameronRP) [inline cptv/reader.py:209]: Would be good to add a print here showing the unknown field and value.

File: cptv/writer.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+            linear = np.arange(width * height, dtype='I')
+            self.twisted = linear + \
+                ((linear // width) & 1) * (width - 1 - 2 * (linear % width))
+        return self.twisted
+
+    def _next_frame(self, pix):
+        linear_pix = np.empty(COLS * ROWS, dtype='h')
+
+        twisted = self._get_twisted()
+        linear_pix = pix.ravel().astype('h')[twisted]
```
Comment (CameronRP) [inline cptv/writer.py]: Is `linear_pix` here not being reassigned before it is used?

**Review by CameronRP — APPROVED** [review]

File: cptv/writer.py (hunk 2 above)
Comment (mjs) [reply cptv/writer.py]: Yeah, that looks wrong.

File: cptv/reader.py (hunk 1 above)
Comment (mjs) [reply cptv/reader.py:209]: Good call. I'll add some output using the `logging` package so that the output can be controlled

**Pull request merged**